### PR TITLE
Adjust a schedule to run workflows in subgroups

### PR DESCRIPTION
**Pull request by magicgoody**
- Subgroups of PreCommit workflows run every 15 minutes for 3 hours, every 6 hours.
- Subgroups of PostCommit workflows run every 15 minutes for 3 hours, every 6 hours.
- Subgroups of PerformanceTests workflows run every 40 minutes for 4 hours, every 12 hours.
- Subgroups of LoadTests workflows run every 40 minutes for 5 hours, every 24 hours.

**PreCommit subgroup consists of 7 workflows.
PostCommit subgroup consists of 6-9 workflows.
PerformanceTests subgroup consists of 4-5 workflows.
LoadTests subgroup consists of 4-5 workflows.**

PreCommit group cron list:

**Example**

[0 1/6 * * *](https://crontab.guru/#0_1/6_*_*_*)
 at 2023-10-18 07:00:00
then at 2023-10-18 13:00:00
then at 2023-10-18 19:00:00
then at 2023-10-19 01:00:00
then at 2023-10-19 07:00:00

[30 5/6 * * *](https://crontab.guru/#30_5/6_*_*_*)
 at 2023-10-19 11:30:00
then at 2023-10-19 17:30:00
then at 2023-10-19 23:30:00
then at 2023-10-20 05:30:00
then at 2023-10-20 11:30:00

Each 6 hours

- 0 1/6 * * *
beam_PreCommit_CommunityMetrics.yml
beam_PreCommit_Go.yml
beam_PreCommit_GoPortable.yml
beam_PreCommit_GoPrism.yml
beam_PreCommit_ItFramework.yml
beam_PreCommit_Java_Amazon-Web-Services_IO_Direct.yml
beam_PreCommit_Java_Amazon-Web-Services2_IO_Direct.yml

- 15 1/6 * * *
beam_PreCommit_Java_Amqp_IO_Direct.yml
beam_PreCommit_Java_Azure_IO_Direct.yml
beam_PreCommit_Java_Cassandra_IO_Direct.yml
beam_PreCommit_Java_Cdap_IO_Direct.yml
beam_PreCommit_Java_Clickhouse_IO_Direct.yml
beam_PreCommit_Java_Csv_IO_Direct.yml
beam_PreCommit_Java_Debezium_IO_Direct.yml

- 30 1/6 * * *
beam_PreCommit_Java_ElasticSearch_IO_Direct.yml
beam_PreCommit_Java_Examples_Dataflow_Java11.yml
beam_PreCommit_Java_Examples_Dataflow_Java17.yml
beam_PreCommit_Java_Examples_Dataflow.yml
beam_PreCommit_Java_File-schema-transform_IO_Direct.yml
beam_PreCommit_Java_Flink_Versions.yml
beam_PreCommit_Java_GCP_IO_Direct.yml

- 45 1/6 * * *
beam_PreCommit_Java_Hadoop_IO_Direct.yml
beam_PreCommit_Java_HBase_IO_Direct.yml
beam_PreCommit_Java_HCatalog_IO_Direct.yml
beam_PreCommit_Java_InfluxDb_IO_Direct.yml
beam_PreCommit_Java_JDBC_IO_Direct.yml
beam_PreCommit_Java_Jms_IO_Direct.yml
beam_PreCommit_Java_Kafka_IO_Direct.yml

- 0 2/6 * * *
beam_PreCommit_Java_Kinesis_IO_Direct.yml
beam_PreCommit_Java_Kudu_IO_Direct.yml
beam_PreCommit_Java_MongoDb_IO_Direct.yml
beam_PreCommit_Java_Mqtt_IO_Direct.yml
beam_PreCommit_Java_Neo4j_IO_Direct.yml
beam_PreCommit_Java_Parquet_IO_Direct.yml
beam_PreCommit_Java_Pulsar_IO_Direct.yml

- 15 2/6 * * *
beam_PreCommit_Java_PVR_Flink_Batch.yml
beam_PreCommit_Java_PVR_Flink_Docker.yml
beam_PreCommit_Java_RabbitMq_IO_Direct.yml
beam_PreCommit_Java_Redis_IO_Direct.yml
beam_PreCommit_Java_SingleStore_IO_Direct.yml
beam_PreCommit_Java_Snowflake_IO_Direct.yml
beam_PreCommit_Java_Solr_IO_Direct.yml

- 30 2/6 * * *
beam_PreCommit_Java_Spark3_Versions.yml
beam_PreCommit_Java_Splunk_IO_Direct.yml
beam_PreCommit_Java_Thrift_IO_Direct.yml
beam_PreCommit_Java_Tika_IO_Direct.yml
beam_PreCommit_Java.yml
beam_PreCommit_Kotlin_Examples.yml
beam_PreCommit_Portable_Python.yml

- 45 2/6 * * *
beam_PreCommit_Python_Coverage.yml
beam_PreCommit_Python_Dataframes.yml
beam_PreCommit_Python_Examples.yml
beam_PreCommit_Python_Integration.yml
beam_PreCommit_Python_PVR_Flink.yml
beam_PreCommit_Python_Runners.yml
beam_PreCommit_Python_Transforms.yml

- 0 3/6 * * *
beam_PreCommit_Python.yml
beam_PreCommit_PythonDocker.yml
beam_PreCommit_PythonDocs.yml
beam_PreCommit_PythonFormatter.yml
beam_PreCommit_PythonLint.yml
beam_PreCommit_RAT.yml
beam_PreCommit_Spotless.yml

- 15 3/6 * * *
beam_PreCommit_SQL_Java11.yml
beam_PreCommit_SQL_Java17.yml
beam_PreCommit_SQL.yml
beam_PreCommit_Typescript.yml
beam_PreCommit_Website_Stage_GCS.yml
beam_PreCommit_Website.yml
beam_PreCommit_Whitespace.yml

***

PostCommit group cron list:

Each 6 hours

- 30 3/6 * * *
beam_PostCommit_Go_Dataflow_ARM.yml
beam_PostCommit_Go_VR_Flink.yml
beam_PostCommit_Go_VR_Samza.yml
beam_PostCommit_Go_VR_Spark.yml
beam_PostCommit_Go.yml
beam_PostCommit_Java_Avro_Versions.yml
beam_PostCommit_Java_BigQueryEarlyRollout.yml
beam_PostCommit_Java_DataflowV1.yml
beam_PostCommit_Java_DataflowV2.yml

- 45 3/6 * * *
beam_PostCommit_Java_Examples_Dataflow_ARM.yml
beam_PostCommit_Java_Examples_Dataflow_Java.yml
beam_PostCommit_Java_Examples_Dataflow_V2_Java.yml
beam_PostCommit_Java_Examples_Dataflow_V2.yml
beam_PostCommit_Java_Examples_Dataflow.yml
beam_PostCommit_Java_Examples_Direct.yml
beam_PostCommit_Java_Examples_Flink.yml
beam_PostCommit_Java_Examples_Spark.yml
beam_PostCommit_Java_Hadoop_Versions.yml

- 0 4/6 * * *
beam_PostCommit_Java_InfluxDbIO_IT.yml
beam_PostCommit_Java_IO_Performance_Tests.yml
beam_PostCommit_Java_Jpms_Dataflow_Java11.yml
beam_PostCommit_Java_Jpms_Dataflow_Java17.yml
beam_PostCommit_Java_Jpms_Direct_Java11.yml
beam_PostCommit_Java_Jpms_Direct_Java17.yml
beam_PostCommit_Java_Jpms_Flink_Java11.yml
beam_PostCommit_Java_Jpms_Spark_Java11.yml
beam_PostCommit_Java_Nexmark_Dataflow_V2_Java.yml

- 15 4/6 * * *
beam_PostCommit_Java_Nexmark_Dataflow_V2.yml
beam_PostCommit_Java_Nexmark_Dataflow.yml
beam_PostCommit_Java_Nexmark_Direct.yml
beam_PostCommit_Java_Nexmark_Flink.yml
beam_PostCommit_Java_Nexmark_Spark.yml
beam_PostCommit_Java_PVR_Flink_Streaming.yml
beam_PostCommit_Java_PVR_Samza.yml
beam_PostCommit_Java_PVR_Spark_Batch.yml
beam_PostCommit_Java_PVR_Spark3_Streaming.yml

- 30 4/6 * * *
beam_PostCommit_Java_Sickbay.yml
beam_PostCommit_Java_SingleStoreIO_IT.yml
beam_PostCommit_Java_Tpcds_Dataflow.yml
beam_PostCommit_Java_Tpcds_Flink.yml
beam_PostCommit_Java_Tpcds_Spark.yml
beam_PostCommit_Java_ValidatesRunner_Dataflow_JavaVersions.yml
beam_PostCommit_Java_ValidatesRunner_Dataflow_Streaming.yml
beam_PostCommit_Java_ValidatesRunner_Dataflow.yml
beam_PostCommit_Java_ValidatesRunner_Direct_JavaVersions.yml

- 45 4/6 * * *
beam_PostCommit_Java_ValidatesRunner_Direct.yml
beam_PostCommit_Java_ValidatesRunner_Flink_Java11.yml
beam_PostCommit_Java_ValidatesRunner_Flink.yml
beam_PostCommit_Java_ValidatesRunner_Samza.yml
beam_PostCommit_Java_ValidatesRunner_Spark_Java11.yml
beam_PostCommit_Java_ValidatesRunner_Spark.yml
beam_PostCommit_Java_ValidatesRunner_SparkStructuredStreaming.yml
beam_PostCommit_Java_ValidatesRunner_Twister2.yml
beam_PostCommit_Java_ValidatesRunner_ULR.yml

- 0 5/6 * * *
beam_PostCommit_Java.yml
beam_PostCommit_Javadoc.yml
beam_PostCommit_PortableJar_Flink.yml
beam_PostCommit_PortableJar_Spark.yml
beam_PostCommit_Python_Arm.yml
beam_PostCommit_Python_Examples_Dataflow.yml
beam_PostCommit_Python_Examples_Direct.yml
beam_PostCommit_Python_Examples_Flink.yml
beam_PostCommit_Python_Examples_Spark.yml

- 15 5/6 * * *
beam_PostCommit_Python_MongoDBIO_IT.yml
beam_PostCommit_Python_Nexmark_Direct.yml
beam_PostCommit_Python_ValidatesContainer_Dataflow_With_RC.yml
beam_PostCommit_Python_ValidatesContainer_Dataflow.yml
beam_PostCommit_Python_ValidatesRunner_Dataflow.yml
beam_PostCommit_Python_ValidatesRunner_Flink.yml
beam_PostCommit_Python_ValidatesRunner_Samza.yml
beam_PostCommit_Python_ValidatesRunner_Spark.yml
beam_PostCommit_Python_Xlang_Gcp_Dataflow.yml

- 30 5/6 * * *
beam_PostCommit_Python_Xlang_Gcp_Direct.yml
beam_PostCommit_Python_Xlang_IO_Dataflow.yml
beam_PostCommit_Python.yml
beam_PostCommit_SQL.yml
beam_PostCommit_TransformService_Direct.yml
beam_PostCommit_Website_Publish.yml
beam_PostCommit_Website_Test.yml
beam_PostCommit_XVR_Direct.yml
beam_PostCommit_XVR_Flink.yml

- 45 5/6 * * *
beam_PostCommit_XVR_GoUsingJava_Dataflow.yml
beam_PostCommit_XVR_JavaUsingPython_Dataflow.yml
beam_PostCommit_XVR_PythonUsingJava_Dataflow.yml
beam_PostCommit_XVR_PythonUsingJavaSQL_Dataflow.yml
beam_PostCommit_XVR_Samza.yml
beam_PostCommit_XVR_Spark3.yml

Each 8 hours
- 30 6/8 * * *
beam_PostCommit_Java_ValidatesRunner_Dataflow_V2_Streaming.yml
beam_PostCommit_Java_ValidatesRunner_Dataflow_V2.yml

Each 24 hours
- 0 8 * * *
beam_PostCommit_Sickbay_Python.yml

***

PerformanceTests group cron list:
40 minutes interval between next running group.

Each 12hours
- 10 9/12 * * *
beam_PerformanceTests_AvroIOIT_HDFS.yml
beam_PerformanceTests_AvroIOIT.yml
beam_PerformanceTests_BigQueryIO_Batch_Java_Avro.yml
beam_PerformanceTests_BigQueryIO_Batch_Java_Json.yml

- 50 9/12 * * *
beam_PerformanceTests_BigQueryIO_Streaming_Java.yml
beam_PerformanceTests_Cdap.yml
beam_PerformanceTests_Compressed_TextIOIT_HDFS.yml
beam_PerformanceTests_Compressed_TextIOIT.yml

- 10 10/12 * * *
beam_PerformanceTests_HadoopFormat.yml
beam_PerformanceTests_JDBC.yml
beam_PerformanceTests_Kafka_IO.yml
beam_PerformanceTests_ManyFiles_TextIOIT_HDFS.yml
beam_PerformanceTests_ManyFiles_TextIOIT.yml

- 50 10/12 * * *
beam_PerformanceTests_MongoDBIO_IT.yml
beam_PerformanceTests_ParquetIOIT_HDFS.yml
beam_PerformanceTests_ParquetIOIT.yml

- 10 11/12 * * *
beam_PerformanceTests_SparkReceiver_IO.yml
beam_PerformanceTests_SQLBigQueryIO_Batch_Java.yml
beam_PerformanceTests_TextIOIT_HDFS.yml
beam_PerformanceTests_TextIOIT.yml

- 50 11/12 * * *
beam_PerformanceTests_TFRecordIOIT_HDFS.yml
beam_PerformanceTests_TFRecordIOIT.yml
beam_PerformanceTests_XmlIOIT_HDFS.yml
beam_PerformanceTests_XmlIOIT.yml

Each 24 hours
- 30 9 * * *
beam_PerformanceTests_BiqQueryIO_Read_Python.yml
beam_PerformanceTests_BiqQueryIO_Write_Python_Batch.yml

- 30 10 * * *
beam_PerformanceTests_PubsubIOIT_Python_Streaming.yml
beam_PerformanceTests_SpannerIO_Read_2GB_Python.yml

- 30 11 * * *
beam_PerformanceTests_SpannerIO_Write_2GB_Python_Batch.yml
beam_PerformanceTests_WordCountIT_PythonVersions.yml
beam_PerformanceTests_xlang_KafkaIO_Python.yml

***

LoadTests group cron list:
40 minutes interval between next running group.

Each 24 hours
- 10 12 * * *
beam_Java_LoadTests_Combine_Smoke.yml
beam_LoadTests_Go_CoGBK_Dataflow_Batch.yml
beam_LoadTests_Go_CoGBK_Flink_batch.yml
beam_LoadTests_Go_Combine_Dataflow_Batch.yml
beam_LoadTests_Go_Combine_Flink_Batch.yml

- 50 12 * * *
beam_LoadTests_Go_GBK_Dataflow_Batch.yml
beam_LoadTests_Go_GBK_Flink_Batch.yml
beam_LoadTests_Go_ParDo_Dataflow_Batch.yml
beam_LoadTests_Go_ParDo_Flink_Batch.yml
beam_LoadTests_Go_SideInput_Dataflow_Batch.yml

- 10 13 * * *
beam_LoadTests_Go_SideInput_Flink_Batch.yml
beam_LoadTests_Java_CoGBK_Dataflow_Batch.yml
beam_LoadTests_Java_CoGBK_Dataflow_Streaming.yml
beam_LoadTests_Java_CoGBK_Dataflow_V2_Batch_JavaVersions.yml
beam_LoadTests_Java_CoGBK_Dataflow_V2_Streaming_JavaVersions.yml

- 50 13 * * *
beam_LoadTests_Java_CoGBK_SparkStructuredStreaming_Batch.yml
beam_LoadTests_Java_Combine_Dataflow_Batch.yml
beam_LoadTests_Java_Combine_Dataflow_Streaming.yml
beam_LoadTests_Java_Combine_SparkStructuredStreaming_Batch.yml
beam_LoadTests_Java_GBK_Dataflow_Batch.yml

- 10 14 * * *
beam_LoadTests_Java_GBK_Dataflow_Streaming.yml
beam_LoadTests_Java_GBK_Dataflow_V2_Batch_Java11.yml
beam_LoadTests_Java_GBK_Dataflow_V2_Batch_Java17.yml
beam_LoadTests_Java_GBK_Dataflow_V2_Streaming_Java11.yml
beam_LoadTests_Java_GBK_Dataflow_V2_Streaming_Java17.yml

- 50 14 * * *
beam_LoadTests_Java_GBK_SparkStructuredStreaming_Batch.yml
beam_LoadTests_Java_ParDo_Dataflow_Batch.yml
beam_LoadTests_Java_ParDo_Dataflow_Streaming.yml
beam_LoadTests_Java_ParDo_Dataflow_V2_Batch_JavaVersions.yml
beam_LoadTests_Java_ParDo_Dataflow_V2_Streaming_JavaVersions.yml

- 10 15 * * *
beam_LoadTests_Java_ParDo_SparkStructuredStreaming_Batch.yml
beam_LoadTests_Python_CoGBK_Dataflow_Batch.yml
beam_LoadTests_Python_CoGBK_Dataflow_Streaming.yml
beam_LoadTests_Python_CoGBK_Flink_Batch.yml
beam_LoadTests_Python_Combine_Dataflow_Batch.yml

- 50 15 * * *
beam_LoadTests_Python_Combine_Dataflow_Streaming.yml
beam_LoadTests_Python_Combine_Flink_Batch.yml
beam_LoadTests_Python_Combine_Flink_Streaming.ym
beam_LoadTests_Python_GBK_Dataflow_Batch.yml

- 10 16 * * *
beam_LoadTests_Python_GBK_Dataflow_Streaming.yml
beam_LoadTests_Python_GBK_Flink_Batch.yml
beam_LoadTests_Python_GBK_reiterate_Dataflow_Batch.yml
beam_LoadTests_Python_GBK_reiterate_Dataflow_Streaming.yml
beam_LoadTests_Python_ParDo_Dataflow_Batch.yml

- 50 16 * * *
beam_LoadTests_Python_ParDo_Dataflow_Streaming.yml
beam_LoadTests_Python_ParDo_Flink_Batch.yml
beam_LoadTests_Python_ParDo_Flink_Streaming.yml
beam_LoadTests_Python_SideInput_Dataflow_Batch.yml

Each 6 hours
- 0 */6 * * *
beam_LoadTests_Python_FnApiRunner_Microbenchmark.yml

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Review by andreydevyatkin — APPROVED** [review]
LGTM

File: .github/workflows/beam_PostCommit_Website_Test.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ name: PostCommit Website Test
 
 on:
   schedule:
-    - cron: '0 */6 * * *'
+    - cron: '30 8/6 * * *'
```
[the remark this answers is not in the document]
Comment (magicgoody) [reply .github/workflows/beam_PostCommit_Website_Test.yml:20]: image
According to VScode, this shows that the job will run every 6 hours. I think there is a possibility that cron will not start at 2:30, according to the [grontab.guru](https://crontab.guru/#30_8/6_*_*_*) website.

***
Re-checked cron on [crontab.guru](https://crontab.guru/#0_5/6_*_*_*) 
'* X/6 * * *' , X should be < 6, so cron will run every 6 hours as intended.

image

[the remark this answers is not in the document]
Comment (magicgoody) [reply .github/workflows/beam_PostCommit_Website_Test.yml:20]: I will update cron schedules

[the remark this answers is not in the document]
Comment (magicgoody) [reply .github/workflows/beam_PostCommit_Website_Test.yml:20]: Fixed

**Review by Abacn — APPROVED** [review]

**Pull request merged**